Gems/MiniAudio/Code/Include/MiniAudio/MiniAudioPlaybackBus.h

```diff
@@ -31,6 +32,24 @@ namespace MiniAudio
         virtual void SetSoundAssetRef(const SoundAssetRef& soundAssetRef) = 0;
         //! Custom getter for scripting
         virtual SoundAssetRef GetSoundAssetRef() const = 0;
+
+        //! Cone controls for directional attenuation
+        virtual float GetInnerAngleInRadians() const = 0;
+        virtual void SetInnerAngleInRadians(float innerAngleInRadians) = 0;
+        virtual float GetInnerAngleInDegrees() const = 0;
+        virtual void SetInnerAngleInDegrees(float innerAngleInDegrees) = 0;
+        virtual float GetOuterAngleInRadians() const = 0;
+        virtual void SetOuterAngleInRadians(float outerAngleInRadians) = 0;
+        virtual float GetOuterAngleInDegrees() const = 0;
+        virtual void SetOuterAngleInDegrees(float outerAngleInDegrees) = 0;
+        virtual float GetOuterVolume() const = 0;
+        virtual void SetOuterVolume(float outerVolume) = 0;
+        virtual bool GetFixedDirecion() const = 0;
+        virtual void SetFixedDirecion(const bool& fixedDirection) = 0;
```

Comment:
This doesn't need to be a const ref because books are simple types like interested and floats

Reply:
See cb1c948747a07a49948c69b7f6480bcf93fec4dc